Assets/Python/Victory.py

```diff
@@ -2811,6 +2811,16 @@ def countFirstDiscovered(iPlayer, iEra):
 			iCount += 1
 	return iCount
 	
+def isFirstDiscoveredPossible(iPlayer, iEra, iRequired):

```

Comment:
Wouldn't be better to just count techs first discovered by `iPlayer` and all undiscovered techs `countFirstDiscovered(-1, iEra)` and see if they add up to at least `iRequired`?

Reply:
Yeah. I'll change it.